### PR TITLE
chore(modules): update trivy to the tip of main

### PR DESCRIPTION
**Pull request by andaaron**
Includes https://github.com/aquasecurity/trivy/pull/5273/commits/ce89d083453f2293d8176c2ac9f4efa0fdf68095

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -44,7 +44,7 @@ require (
 )
 
 require (
-	github.com/aquasecurity/trivy v0.45.1
+	github.com/aquasecurity/trivy v0.45.2-0.20231009110409-91841f59bac1
```
Comment (andaaron) [inline go.mod:47]: Note this is a pseudo-tag, as Trivy has not released wit the fix yet.

**Review by peusebiu — APPROVED** [review]

**Review by rchincha — APPROVED** [review]
lgtm

Let's keep an eye on an official release.

**Comment by rchincha** [comment]
Conflicts?

**Review by rchincha — APPROVED** [review]
lgtm

**Pull request merged**